modules/extensions/apps/portal/libs/pc-analytics-libs/css/custom.css

```diff
@@ -0,0 +1,23 @@
+.fw-add:before {
+    content: '\e615';
+}
+
+.panel-body a {
+    color: rgb(52, 59, 89);
+    text-decoration: none;
+}
+
+.fw-filter:before {
+    content: '\e634';
+}
+
+/*a:hover{*/
+    /*text-decoration: none;*/
+/*}*/
+
+@media (min-width: 400px) {
+    .form-inline .form-control {
+        display: inline-block;
+        width: auto;
+        vertical-align: middle;  }
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7